Makefile

```diff
@@ -25,8 +25,9 @@ BUILD_OMS_VAL=0
 endif
 
 all:
-	rm -rf release/*.{rpm,deb};
+	rm -rf release/*.rpm release/*.deb
```

Comment:
Historically for the projects that I have worked on for the OSTC team, we have rebuilt packages every time make was run.  This is in keeping with precedent, although it is arguable whether this should continue to be done.  Building packages for dsc doesn't take very long at all compared to some of our other projects, so for now I will not refactor this